src/Domain/Tags/FaceNormal.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Domain/Tags.hpp"
+
+namespace domain::Tags {
+
+/// The _normalized_ face normal
+template <size_t Dim, typename LocalFrame = Frame::Inertial>
+using FaceNormal =
+    ::Tags::Normalized<Tags::UnnormalizedFaceNormal<Dim, LocalFrame>>;
+
+/// The magnitude of the _unnormalized_ face normal, see
+/// `::unnormalized_face_normal`
+template <size_t Dim, typename LocalFrame = Frame::Inertial>
+using FaceNormalMagnitude =
```

Comment:
`UnnormalizedFaceNormalMagnitude` or something.  `FaceNormalMagnitude` sounds too much like the magnitude of `FaceNormal`.

Reply:
I tried to keep this name short, so I state explicitly this is the magnitude of the _unnormalized_ face normal in the docstring, and also link to `unnormalized_face_normal` for the definition. I also thought there's little chance of mis-interpreting this because the magnitude of the _normalized_ face normal is obviously just 1..

Reply:
Let's see if the second reviewer has any input.

Reply:
ok fine